docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Introduction
+
+The purpose of this document is to show contributors to the modernisation platform code the best practises and testing that should be undertaken. Code should be in Terraform and tests, if coded, in Go.
+
+## Code of Conduct
+
+The Ministry of Justice [Code of Conduct](https://github.com/ministryofjustice/.github/blob/main/CODE_OF_CONDUCT.md) is used to govern the project and those participating.
+Please report unacceptable behavior to the maintainers or administrators.
+
+## Standards
+
+We follow the [MoJ Technical Guidance](https://technical-guidance.service.justice.gov.uk/#moj-technical-guidance),
+which complements the Service Standard,
+and also the MoJ [Cyber and Technical Security Guidance](https://security-guidance.service.justice.gov.uk/#cyber-and-technical-security-guidance).
+
+# Modernisation Platform Documentation
+To access the modernisation documentation follow this [MP Documentation](https://user-guide.modernisation-platform.service.justice.gov.uk/#modernisation-platform).
+For the general terraform you can access here [Terraform](https://user-guide.modernisation-platform.service.justice.gov.uk/runbooks/terraform.html)
+If you are interested in running Terraform locally try [Running Terraform Plan Locally](https://user-guide.modernisation-platform.service.justice.gov.uk/user-guide/running-terraform-plan-locally.html#running-terraform-plan-locally)
+
+These can be located by running a search in the documentation [MP Documentation](https://user-guide.modernisation-platform.service.justice.gov.uk/#modernisation-platform). 
+
+# Coding examples
+
+There are a number of examples in this repository of coding and also addutional ones in the modernisation-platform-environments repository. One are we recommend users exmaine is the example code in moderisation-platform-environments/terrafor/environments//example area. This has sameples of code showing how to build and ec2, security groups, load blanacnders, rds and others. These are named well and the code can be copied and pasted into your own code so they are available locally. Obviously when these are copied the names should be changed so they do not contain the example names.
```

Comment:
Not sure if I would mention this as the coding examples relate to user code rather than module code, unless you make the suggestion that they update the example code with any changes they make to the module?

Reply:
It doesn't check spelling :-( 
I will include your other suggestion.